java-servlet-json/src/main/webapp/js/myfunctions.js

```diff
@@ -1,50 +1,59 @@
 function sendAjax() {
- 
-	// get inputs
-	var article = new Object();
-	article.title = $('#title').val();
-	article.url = $('#url').val();
-	article.categories = $('#categories').val().split(";");
-	article.tags = $('#tags').val().split(";");
-	
-	$.ajax({
-		url: "jsonservlet",
-		type: 'POST',
-		dataType: 'json',
-		data: JSON.stringify(article),
-		contentType: 'application/json',
-		mimeType: 'application/json',
-		
-		success: function (data) {
-        	$("tr:has(td)").remove();
-
-        	$.each(data, function (index, article) {
-            	
-                var td_categories = $("<td/>");
-                $.each(article.categories, function (i, tag) {
-                	var span = $("<span class='label label-info' style='margin:4px;padding:4px' />");
-                	span.text(tag);
-                	td_categories.append(span);
-                });
-                
-                var td_tags = $("<td/>");
-                $.each(article.tags, function (i, tag) {
-                	var span = $("<span class='label' style='margin:4px;padding:4px' />");
-                	span.text(tag);
-                	td_tags.append(span);
-                });
-                
-                $("#added-articles").append($('<tr/>')
-                		.append($('<td/>').html("<a href='"+article.url+"'>"+article.title+"</a>"))
-                		.append(td_categories)
-                		.append(td_tags)
-                );
-            
-                
-            }); 
+
+    // get inputs
+    var article = new Object();
+    article.title = $('#title').val();
+    article.url = $('#url').val();
+    article.categories = $('#categories').val().split(";");
+    article.tags = $('#tags').val().split(";");
+
+    $.ajax({
+        url: "jsonservlet",
+        type: 'POST',
+        dataType: 'json',
+        data: JSON.stringify(article),
+        contentType: 'application/json;charset=utf-8',
+        mimeType: 'application/json;charset=utf-8',
+
+        success: function (data) {
+            drawArticles(data);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["drawArticles" was used before it was defined](https://www.codacy.com/app/maratmingazovr/Java/file/2724704929/issues/source?bid=3348317&fileBranchId=3348327#l19)